chain/network/src/concurrency/demux.rs

```diff
@@ -126,6 +132,9 @@ impl<Arg: 'static + Send, Res: 'static + Send> Demux<Arg, Res> {
     pub fn new(rl: RateLimit) -> Demux<Arg, Res> {
         rl.validate().unwrap();
         let (send, mut recv): (Stream<Arg, Res>, _) = mpsc::unbounded_channel();
+        // TODO(gprusak): this task should be running as long as Demux object exists.
+        // "Current" runtime can have a totally different lifespan, so we shouldn't spawn on it.
+        // Find a way to express "runtime lifetime > Demux lifetime".
```

Comment:
A typical way to do this is to have a single runtime which lives as long as the application itself. I am actually not sure if actix creates a runtime per arbiter or uses a single one.

Reply:
It has a runtime per arbiter